src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/CreateApprenticeshipRequestValidatorBase.cs

```diff
@@ -1,19 +1,20 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using System.Linq.Expressions;
 using FluentValidation;
 using FluentValidation.Internal;
 
 namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator
 {
     public abstract class CreateApprenticeshipRequestValidatorBase
     {
-        protected ValidationContext<T> GetValidationContextForProperty<T>(T objectToValidate, string propertyNameToValidate)
+        protected ValidationContext<T> GetValidationContextForProperty<T>(T objectToValidate, string propertyName)
```

Comment:
What was wrong with 'propertyNameToValidate'?

Reply:
We could probably delete this function and update the other tests to use the Expression instead

Reply:
I suggest we only delete after this one is on master. Then we're sure of no conflicts - new  stories after this is on master can use it. All previous don't which can be fixed as a tech debt story.